README.md

```diff
@@ -18,14 +18,16 @@
 
 # Table Of Contents
 * [Developing](#developing)
+   * [Contribution Guidelines](#contribution-guidelines)
    * [Requirements](#requirements)
    * [Building](#building)
 * [Using Conan with Nexus Repository Manager 3](#using-conan-with-nexus-repository-manager-3)
 * [Installing the plugin](#installing-the-plugin)
+   * [Permanent Install](#permanent-reinstall)
```

Comment:
Minor/Q: Should this say "Permanent Reinstall"?  I only noticed it's the same text as "Permanent Install" text below.

Reply:
fixed 9256cd94a7e43a6f43ad6f942b7665f0468ccb7d